smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -198,14 +238,15 @@ contract DigitalaxAuction is Context, ReentrancyGuard {
      */
     function resultAuction(uint256 _garmentTokenId) external nonReentrant {
 
-        // TODO check this should be admin and smart contract role check
         // Admin only resulting function
-        require(accessControls.hasAdminRole(_msgSender()), "DigitalaxAuction.resultAuction: Sender must be admin");
+        require(
+            accessControls.hasAdminRole(_msgSender()) || accessControls.hasSmartContractRole(_msgSender()),
+            "DigitalaxAuction.resultAuction: Sender must be admin"
```

Comment:
must be admin or whitelisted smart contract

Reply:
fixing